tests/test_messages_config/test_messages_config.py

```diff
@@ -81,3 +81,46 @@ def test_no_message_config_and_no_pyta_overwrite():
 
     assert "This custom error message is modified." not in output.stdout
     assert "The first reversed() argument is not a sequence" in output.stdout
+
+
+def test_valid_message_config_no_section_header_and_pyta_overwrite():
```

Comment:
The test cases themselves are good. However, the change you've made to remove the monkeypatching lets us go a step further in simplification.

Previously, `subprocess.run` was used so that the pylint checkers would be reloaded for each test case; this was necessary because PythonTA patched these classes. Now that we're no longer doing this, we can switch to simply using `python_ta.check_all`, which is substantially faster than spawning subprocesses.